pkg/util/util.go

```diff
@@ -436,3 +436,56 @@ func GetSortedKeys(mapping map[string]string) []string {
 
 	return keys
 }
+
+//returns a slice containing the split string, using ',' as a separator
+func GetSplitValuesFromStr(inputStr string) []string {
+	if len(inputStr) == 0 {
+		return []string{}
+	}
+
+	result := strings.Split(inputStr, ",")
+	for i, port := range result {
```

Comment:
I'd rename that `port` variable to something else ;)

Reply:
LOL, isn't port the best possible name?